packages/experiments/src/components/Button/MenuButton/MenuButton.state.ts

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { BaseState } from '../../../utilities/BaseState';
+import { IMenuButtonProps, IMenuButtonViewProps } from './MenuButton.types';
+
+export type IMenuButtonState = Pick<IMenuButtonViewProps, 'expanded' | 'onClick' | 'onKeyDown' | 'onMenuDismiss' | 'menuTarget'>;
+
+export class MenuButtonState extends BaseState<IMenuButtonProps, IMenuButtonViewProps, IMenuButtonState> {
```

Comment:
Unrelated to this PR but one action item we need to do is remove BaseComponentMin from BaseState entirely.

Also unrelated I need to examine BaseState in the context of Hooks. I think it may be deprecated entirely.